katz/src/main/kotlin/katz/data/Kleisli.kt

```diff
@@ -3,14 +3,14 @@ package katz
 typealias KleisiTKind<F, A, B> = HK3<Kleisli.F, F, A, B>
 typealias KleisiF<F> = HK<Kleisli.F, F>
```

Comment:
I believe these two are misnamed too. Could you please use this diff to rename them to Kleisli?